app/src/main/java/at/roteskreuz/stopcorona/model/repositories/ReportingRepository.kt

```diff
@@ -177,7 +177,6 @@ class ReportingRepositoryImpl(
                 .startOfTheUtcDay()
```

Comment:
`thresholdTime` seems unused

Reply:
and `uploadKeysDays` and `configuration`

Reply:
updated